python/src/opendp/mod.py

```diff
@@ -258,6 +260,126 @@ def __iter__(self):
         # this overrides the implementation of __iter__ on POINTER, 
         # which yields infinitely on zero-sized types
         raise ValueError("Measurement does not support iteration")  # pragma: no cover
+    
+
+class Odometer(ctypes.POINTER(AnyOdometer)): # type: ignore[misc]
+    """A differentially private unit of computation with no privacy limit.
+    An odometer contains a function that returns a queryable with a function and a relation.
```

Comment:
```suggestion
    An odometer contains a function that returns a queryable with a function and a relation.
```
Should it be `Odometer` and `Queryable`? It feels like you're telling me the type signature (two "functions" and a "relation") without explaining what they do.